src/config/ConfigManager.cpp

```diff
@@ -360,6 +360,7 @@ CConfigManager::CConfigManager() {
     m_pConfig->addConfigValue("group:groupbar:enabled", Hyprlang::INT{1});
     m_pConfig->addConfigValue("group:groupbar:font_family", {STRVAL_EMPTY});
     m_pConfig->addConfigValue("group:groupbar:font_size", Hyprlang::INT{8});
+    m_pConfig->addConfigValue("group:groupbar:group_rounding", Hyprlang::INT{0});
```

Comment:
`group_rounding` instead of just `rounding` seems redundant since it's already in the `group:groupbar` section